### ISSUE TITLE
[LRM] Request for Pre-trained Latent Reward Models (Strategy 1 and Strategy 3)

### ISSUE THREAD

**Issue by LTH9898** [Kwai-Kolors/LPO#9]
Hi, thanks for releasing the code and the pre-trained Strategy 2 Latent Reward Model (LRM).

In the paper, you describe three MPCF strategies (Table 1, Section 4.2), but only Strategy 2 is currently available here. For reproducibility and to replicate the ablation results (e.g., Table 7), could you also provide the Strategy 1 and Strategy 3 pre-trained LRM models?

Thanks again for the great work and open-sourcing the models!

**Comment by casiatao** [comment]
Sorry, we have not retained the LRM corresponding to strategies 1 and 3. You can train the corresponding LRM using the code and data by yourself.
